sqlpage/templates/login.handlebars

```diff
@@ -0,0 +1,89 @@
+<div class="container {{class}}">
+    <div class="row d-flex justify-content-center">
+        <div class="col-md-6">
+            <form 
+                {{#if id}}id="{{id}}"{{/if}}
+                class="mt-3 mb-1" 
+                method="post" 
+                {{#if enctype}}enctype="{{enctype}}"{{/if}}
+                {{#if action}}
+                    action="{{action}}"
+                {{else}}
+                    {{#if id}}action="#{{id}}"{{/if}}
+                {{/if}}
+            >
+                {{#if image}}
+                    <div class="form-group d-flex justify-content-center align-items-center">
+                        <img src="{{image}}"/>
+                    </div>
+                {{/if}}
+                {{#if title}}
+                    <h1 class="text-center mb-3">{{title}}</h1>
+                {{/if}}
+                <label class="form-label">{{username}}</label>    
+                <div class="input-icon mb-3">
+                    <span class="input-icon-addon">
+                        <svg xmlns="http://www.w3.org/2000/svg" class="icon" width="24"
+                            height="24" viewBox="0 0 24 24" stroke-width="2" stroke="currentColor"
+                            fill="none" stroke-linecap="round" stroke-linejoin="round">
+                            <path stroke="none" d="M0 0h24v24H0z" fill="none" />
+                            <circle cx="12" cy="7" r="4" />
+                            <path d="M6 21v-2a4 4 0 0 1 4 -4h4a4 4 0 0 1 4 4v2" />
+                        </svg>
+                    </span>
```

Comment:
use builtin tabler icons like in the other components ? And maybe make the icon configurable ?